dist/ng-material-multilevel-menu/public_api.d.ts

```diff
@@ -1,2 +1,3 @@
 export * from './lib/ng-material-multilevel-menu.module';
+export * from './lib/ng-material-multilevel-menu.component';
```

Comment:
Just curious, why to export the component!

Reply:
Angular 9 requires this. Without exporting the component it won't work.